TELCODOCS#1876: Moved troubleshooting assembly modules to the main LVMS assembly (#75839)
openshift/openshift-docs: modules/lvms-troubleshooting-recovering-from-missing-lvms-or-operator-components.adoc

:_mod-docs-content-type: PROCEDURE
[id="recovering-from-missing-lvms-or-operator-components_{context}"]

> = Recovering from missing StorageClass

Author: Updated as per the suggestion.

> = Recovering from missing storage class

Reviewer: ```suggestion
= Recovering from missing StorageClass
```

Author: Updated as per the suggestion.

vg-manager-r6zdv                      1/1     Running   0             66m
----
+
The output of this command must contain a running instance of the following pods:

* `lvms-operator`
* `vg-manager`

> * `topolvm-controller`
> * `topolvm-node`

Reviewer: ```suggestion
```
This is no longer the case for 4.16 and beyond, but still necessary for lower versions

Author: As there are two updates specific to 4.16+, I will make note of these and create another PR to make these updates only in 4.16.

+

> If the `topolvm-node` pod is stuck in the `Init` state, it is due to a failure to locate an available disk for {lvms} to use. To retrieve the necessary information to troubleshoot this issue, review the logs of the `vg-manager` pod by running the following command:

Reviewer: ```suggestion
If the `vg-manager` pod is stuck trying to load a configuration file, it is due to a failure to locate an available disk for {lvms} to use. To retrieve the necessary information to troubleshoot this issue, review the logs of the `vg-manager` pod by running the following command:
```

This needs to change to check for vg-manager being stuck for 4.16+

Author: As there are two updates specific to 4.16+, I will make note of these and create another PR to make these updates only in 4.16.
